extensions/claude-throne/webview/main.js

```
@@ -165,66 +504,204 @@
+      case 'keysLoaded':
+        handleKeysLoaded(message.payload.keyStatus || message.payload);
+        break;
       case 'keyStored':
         handleKeyStored(message.payload);
         break;
+      case 'anthropicKeyStored':
+        handleAnthropicKeyStored(message.payload);
+        break;
       case 'proxyError':
-        showError(message.payload);
+      case 'proxyError': {
```

Biome (2.1.2)

[error] Duplicate case label.

The first similar label is here:


(lint/suspicious/noDuplicateCase)